aviatrix/data_source_aviatrix_firewall_instance_images.go

```diff
@@ -73,3 +80,140 @@ func dataSourceAviatrixFirewallInstanceImagesRead(d *schema.ResourceData, meta i
 	d.SetId(vpcId)
 	return nil
 }
+
+func sortImageVersion(listVersion []string) []string {
+	var n = len(listVersion)
+	for i := 0; i <= n-1; i++ {
+		for j := i; j <= n-1; j++ {
+			var splitFlag string
+			if checkFirstCharacter(listVersion[i]) == "R" {
```

Comment:
Instead of deciding how to sort by using the first character in the version name, you should probably use the description. That way if we add anther type of versioning that starts with R, but has a different format it won't break .